### DISCUSSION TITLE
How did ambie sideloading install ?

### DISCUSSION THREAD

**Discussion by Strypper** [jenius-apps/ambie#370] [Q&A] [answered]
I thought UWP app doesn't have the ability to do sideloading and also MSIX with Ambie doesn't seem to require any certificate where you purchased the certificate that accepted on public Windows OS?

**Comment by dpaulino** [comment]
Sideloading is unrelated to uwp itself but rather it's related to MSIX. Any properly MSIX package codesigned with a trusted certificate can be installed on machines.

**Reply by Strypper** [reply]
It has to be trusted by the root level and then we can be able to install applications, to fully trust this we need to buy certificates like Comodo?

**Reply by dpaulino** [reply, marked as answer]
yeah
